perun-base/src/test/resources/perun-roles.yml

```diff
@@ -20,4 +20,12 @@ perun_roles:
   - SECURITYADMIN
   - CABINETADMIN
   - UNKNOWN
+
+#A list of Perun policies that are loaded to the perunPolicies map in AuthzResolverImpl
+perun_policies:
```

Comment:
If this stub is required on all deployed instances, then please add it to the perun-ansible repository too.

Reply:
PR was created CESNET/perun-ansible#122